common/testlib.h

```diff
@@ -181,6 +181,7 @@ const char *latestFeatures[] = {
 #include <stdarg.h>
 #include <fcntl.h>
 #include <functional>
+#include <cstdint>
```

Comment:
```suggestion
#include <cstdint> // modified for library-checker
```

一応改変したことを明記しておいてほしいです

Reply:
了解です。提案を取り込みました。